src/wp-includes/class-wp-application-passwords.php

```diff
@@ -284,6 +287,11 @@ public static function update_application_password( $user_id, $uuid, $update = a
 				$save         = true;
 			}
 
+			if ( ! empty( $update['password'] ) && $item['password'] !== $update['password'] ) {
```

Comment:
Is this being used? I'm not seeing it, but I may be missing something.

Reply:
Thanks, this is a remnant of an earlier change where application password hashes were being opportunistically upgraded, but that's no longer happening. I'll remove it.